### PR TITLE
start migrating html to Jest

### LINKED ISSUE #2879: Start migrating HTML to jest

### PR DESCRIPTION
**Pull request by afebbraro**
## What does this PR do?
Migrate HTML to use Jest! This aligns all three of our frameworks. 

### Associated Issue
Fixes #2879 

## Please check off completed items as you work.
If a checklist item or section does not apply to your PR
then please remove it.

### Code
 - [x] Build Component in HTML
 - [x] Unit Testing in HTML with `npm run test` in `html/` (100% coverage, 100% passing)

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)

### CODE REVIEW

File: html/tests/tabs.tests.js [hunk 1]
```diff
@@ -105,74 +101,76 @@ describe('tabs UI Events tests', () => {
...
+    expect(tab3.addEventListener.getCall(0).args[0]).toBe('click');
   });
 
   it('should bind the keydown event to the container', () => {
-    expect(tabContainer.addEventListener.getCall(0).args[0]).eql('keydown');
+    expect(tabContainer.addEventListener.getCall(0).args[0]).toBe('keydown');
   });
 
   it('should activate the previous tab when left is pressed', () => {
     event = new window.Event('keydown');
```
Comment (guern1kn) [inline html/tests/tabs.tests.js]: Should these be `event = new Event('keydown');` as on line 141?

File: html/tests/tabs.tests.js [hunk 2]
```diff
@@ -105,74 +101,76 @@ describe('tabs UI Events tests', () => {
...
     tab2.click();
-    expect(tab2.classList.contains('sprk-c-Tabs__button--active')).eql(true);
+    expect(tab2.classList.contains('sprk-c-Tabs__button--active')).toBe(true);
     tabContainer.dispatchEvent(event);
-    expect(tab1.classList.contains('sprk-c-Tabs__button--active')).eql(true);
+    expect(tab1.classList.contains('sprk-c-Tabs__button--active')).toBe(true);
   });
 
   it('should activate the next tab when right is pressed', () => {
     event = new window.Event('keydown');
```
Comment (guern1kn) [inline html/tests/tabs.tests.js]: `event = new Event('keydown');` ?

File: html/tests/tabs.tests.js [hunk 3]
```diff
@@ -105,74 +101,76 @@ describe('tabs UI Events tests', () => {
...
     tab2.click();
-    expect(tab2.classList.contains('sprk-c-Tabs__button--active')).eql(true);
+    expect(tab2.classList.contains('sprk-c-Tabs__button--active')).toBe(true);
     tabContainer.dispatchEvent(event);
-    expect(tab3.classList.contains('sprk-c-Tabs__button--active')).eql(true);
+    expect(tab3.classList.contains('sprk-c-Tabs__button--active')).toBe(true);
   });
 
   it('should activate the first tab when Home is pressed', () => {
     event = new window.Event('keydown');
```
Comment (guern1kn) [inline html/tests/tabs.tests.js]: event = new Event('keydown');

File: html/tests/tabs.tests.js (hunk 2 above)
Comment (afebbraro) [reply html/tests/tabs.tests.js]: window.Event works since I added it to jest globals, so this way or the other way works.

**Pull request merged**